mscore/musescore.cpp

```diff
@@ -3611,10 +3611,21 @@ AboutBoxDialog::AboutBoxDialog()
 //   copyRevisionToClipboard
 //---------------------------------------------------------
 
+#include <QSysInfo>
```

Comment:
Is this include really needed ? We use QSysInfo elsewhere without it. It compiles on mac without it apparently.

Reply:
No, it is not, not sure why I though it was.